docker/Dockerfile

```diff
@@ -62,8 +62,11 @@ RUN mkdir /var/farmOS \
   && rm -r /opt/drupal && cp -rp /var/farmOS /opt/drupal
 
 # Set the entrypoint.
-COPY docker-entrypoint.sh /usr/local/bin/
-RUN chmod u+x /usr/local/bin/docker-entrypoint.sh
+COPY --chown=www-data docker-entrypoint.sh /usr/local/bin/
```

Comment:
Oh this is neat! I wasn't aware of the `COPY --chown` option! I see that you also changed the mode of the file from `0644` to `0755` in the repo itself. Makes sense! :+1: